provider/equinix/environ_test.go

```diff
@@ -378,6 +381,88 @@ func (s *environProviderSuite) TestStartInstance(c *gc.C) {
 				},
 			},
 		},
+		{
```

Comment:
The test is looking for the g2.large.x86 plan, so to exercise the filters I think it will be necessary to make g2.large.x86 the last plan in this list.

Reply:
Apart from fixing the issue with dashes vs underscores you will also need to add the g2.large.x86 entry to L519 and L528 (exp response from packet `List` endpoint) for the test to work.